test/app.js

```diff
@@ -151,8 +152,18 @@ describe('generator-camel:app', function () {
     });
 
     it('Should create pom.xml with default content', function () {
-      assert.fileContent('pom.xml', new RegExp('<groupId>' + basicProps.package + '</groupId>') );
-      assert.fileContent('pom.xml', new RegExp('<artifactId>' + basicProps.name + '</artifactId>') );
+      assert.fileContent('pom.xml', new RegExp('<groupId>' + basicProps.package + '</groupId>'));
+      assert.fileContent('pom.xml', new RegExp('<artifactId>' + basicProps.name + '</artifactId>'));
+    });
+  });
+
+  describe('Should test the utils class', function () {
+    it('utilities package validation should work for valid package', function () {
+      assert.strictEqual(utils.validatePackage('com.valid'), true);
+    });
+    it('utilities package validation should fail for invalid package', function () {
+      assert.notStrictEqual(utils.validatePackage('invalid@.pkg.name'), true);
+      assert.notStrictEqual(utils.validatePackage('a.name.with.package'), true);
```

Comment:
it is a good practice to have a single assertion per test when it is possible. it allows to provide more precise errors. When grouped in single method, you won't know immediately if both cases are broken or a single one.

Here it is easily possible as the validatePackage doesn't rely on previous validatePackage call.

Reply:
Fixed